tx-source/site_core.php

```diff
@@ -1930,3 +1930,9 @@
 // Quotes/Formats
 $Definition['Toggle Quote'] = 'Toggle Quote';
 $Definition['Spoiler Warning'] = 'Spoiler Warning';
+
+// Catalogue Display Plugin/Formats
+$Definition['You must have the Post and Mobile Formats set to anything but "Text" in the Advanced Editor Plugin.'] = 'You must have the Post and Mobile Formats set to anything but "Text" in the Advanced Editor Plugin.';
+$Definition['Each discussion will show an uploaded image on the Discussions page instead of the author information. This only applies to categories with "Discussions" as the "Display As.'] = 'Each discussion will show an uploaded image on the Discussions page instead of the author information. This only applies to categories with "Discussions" as the "Display As.';
```

Comment:
```suggestion
$Definition['Each discussion will show an uploaded image on the Discussions page instead of the author information. This only applies to categories with "Discussions" as the "Display As.'] = 'Each discussion will show an uploaded image on the Discussions page instead of the author information. This only applies to categories with "Discussions" as the "Display As."';
```

Missing an end quote here. IS this still accurate after the recent changes though?

Reply:
Good call. Old description. I updated it.